config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```
@@ -9,6 +9,7 @@ postsubmits:
       decorate: true
       branches:
         - ^master$
+        - ^release-0.2$
```

If you specifically want just this one branch:

```suggestion
        - ^release-0\.2$
```

But what if we did this instead, to cover all release branches?
```
^release-
```

---

I don't think the referenced script is present in the release-0.1 branch

---

Which script?

---

It's missing the cloud-build.yaml and the make targets used by it.